Lib/venv/__init__.py

```diff
@@ -251,7 +251,7 @@ def setup_python(self, context):
 
             for suffix in suffixes:
                 src = os.path.join(dirname, suffix)
-                if os.path.exists(src):
+                if os.path.lexists(src):
```

Comment:
Because the contents of the `WindowsApps\Python...` folder are symlinks, we need `lexists` here to see them.